src/main/java/com/codeofus/rent_a_park/dtos/PersonDto.java

```diff
@@ -22,9 +22,9 @@ public class PersonDto implements Serializable {
     String lastName;
     @NotNull
     String registration;
-    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
I'd still just delete @JsonInclude... if list is empty client will get a field with empty list, and in Javascript clients for example it's be resolved as empty array/list, but if we don't send the field, then it's undefined which adds another thing to worry about downstream